changelog/db.changelog-1.2.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+ <changeSet author="Simon Dunn" id="1">
+    <tagDatabase tag="v1.1.0" />
+  </changeSet>
+  <changeSet author="Simon Dunn" id="2">
+    <addColumn tableName="settlements">
+      <column name="value" afterColumn="sourceSystem" type="INTEGER" />
+    </addColumn>
+  </changeSet>
```

Comment:
Something we've also decided to do is to DROP the `sourceSystem` column in `settlements`

We are instead going to save `sourceSystem` into `paymentRequests` (i.e. `SFI`)
We are not going to save `sourceSystem` into `settlements` (i.e. `SITIAgri` which is in the message)

Because we _have_ the field in the message we will need an explicit integration test to check that it is *not* in the `settlements` table